### PR TITLE
patch: add requirements and help to issue template

### PR DESCRIPTION
**Pull request by mcraa**

### CODE REVIEW

**Comment by mcraa** [comment]
@balena-ci rebase

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 - **Etcher version:** 
 - **Operating system and architecture:** 
 - **Image flashed:** 
+- **What do you think should have happened:** <!-- even step by step reproduction process -->
```
Comment (lurch) [inline .github/ISSUE_TEMPLATE.md:4]: It's not clear what the "even" means here - perhaps it should be removed? Or reworded to something like "or a step by step reproduction process" ?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+- **What happened:**
 - **Do you see any meaningful error information in the DevTools?** 
-
 <!-- You can open DevTools by pressing `Ctrl+Shift+I` (`Ctrl+Alt+I` for Etcher before v1.3.x), or `Cmd+Opt+I` if you're on macOS. -->
+
+<!-- issues with missing information will be labeled as not-enough-info and closed shortly -->
+<!-- plase try to include as many influencing elements as possible are you root, does any other process block the device, etc. -->
```
Comment (lurch) [inline .github/ISSUE_TEMPLATE.md:10]: "plase" -> "please"? Should probably be a colon after "possible"? And "factors" might be a better word than "elements"?

**Comment by mcraa** [comment]
@balena-ci I self-certify!

**Comment by thgreasi** [comment]
@balena-ci republish

**Pull request merged**